src/Template/Element/top_menu.ctp

```diff
@@ -226,6 +229,32 @@ $sidenavPosition = $htmlDir == 'rtl' ? 'md-sidenav-right' : 'md-sidenav-left';
                 </div>
             </div>
 
+            <md-list>
+                <?php if (!$session->read('Auth.User.id') && !$isOnLoginPage) { ?>
+                <md-list-item href="<?= $registerUrl ?>">
+                    <p>
+                    <md-icon>chevron_right</md-icon>
+                    <?php
+                    /* @translators: link to the Register page in the top bar (verb) */
+                    echo __('Register');
+                    ?>
+                    </p>
+                </md-list-item>
+
+                <md-list-item ng-controller="LoginDialogController as vm" ng-click="vm.showDialog('<?= $this->Pages->currentPageUrl() ?>')">
+                    <p>
+                    <md-icon>chevron_right</md-icon>
+                    <?php 
+                    /* @translators: link to open the Login box in the top bar (verb) */
+                    echo __('Log in');
```

Comment:
These two translators notes are incorrect. We are not in the top bar but the navigation side menu.